.github/workflows/release_shup.yml

```diff
@@ -0,0 +1,84 @@
+name: Java CI to create and upload Shanoir Uploader release on pull request
+on:
+  push:
+    branches: [ "master" ]
+    paths:
+      - 'shanoir-uploader/**'
+  pull_request:
```

Comment:
Hi @LaurentPV, I wonder, if we should do something on each pull request or only, when something goes on master? I think even with each new integration into master, we will not release a new version of ShUp. I think it would be interesting for each push into master to create a new "timestamp" in RELEASE_DATE on ShUpConfig.
I think the process of having a new version, as today the tags for the prod release by Anthony are driven by a human decision, not by an integration on master. With kind regards, Michael

Reply:
Ok, I will add a new commit modifying the RELEASE_DATE constant with the timestamp of the push on master branch. 
Do you mean that the tag of the release should be the release_date instead of the release_version ? 
It seems weird not to have the tag of a version linked to a release but in the same time it would be more convenient because it assures the uniqueness of each release tag. For now if you forget to modify the version in the pom.xml before pushing into master the release will fail because the previous release exists already.